README.md

```diff
@@ -1 +1,42 @@
-# vocabulary
\ No newline at end of file
+
+## IDR Controlled Vocabularies and Ontology Annotation
+
+We are using a number of controlled vocabularies and ontologies in the BBSRC-funded [Image Data Repository](http://idr-demo.openmicroscopy.org/webclient/userdata/?experimenter=-1) (IDR) project.  Our approaches to annotation are being modified and expanded as we gather more imaging datasets, expand the functionality we are providing within OMERO for the IDR,  and as a result of discussions with others about the best approaches to take.
+
+
+We have a spreadsheet outlining the [controlled vocabulary and ontology sources](https://docs.google.com/spreadsheets/d/1S9of23dD8vY1QUv90RV_-Ugu0h6yTeNobuj92-OoSl8/edit?usp=sharing) for terms we are currently using for High Content Screens (HCS).
+
+
+### Dataset descriptors
+
+For high level information describing the type of dataset, e.g. "this is a primary RNAi High Content Screen imaging dataset",  we have put together some terms to describe aspects of the High Content Screens and the protocols describing the steps in the screen.  These terms are in the [Experimental Factor Ontology](http://www.ebi.ac.uk/efo) (EFO).
+
+
+For example the following terms (and child terms below them) have been added:
+
+ * [high content screen](http://www.ebi.ac.uk/ols/beta/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007550)
+ * [high content screening stage](http://www.ebi.ac.uk/ols/beta/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007555)
+ * [screening library](http://www.ebi.ac.uk/ols/beta/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007559)
+ * [HCS protocols](http://www.ebi.ac.uk/ols/beta/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0007570)
+
+We are also looking at using terms from the [Biological imaging methods](http://www.ebi.ac.uk/ols/beta/ontologies/fbbi) ontology or importing them into EFO.
+
+
+### Samples, Experimental Variables and Library Annotation
+
+For sample annotation we are using [NCBITaxon](http://www.ebi.ac.uk/ols/beta/ontologies/ncbitaxon) for organisms, and EFO so far for cell lines.  EFO should also cover most sample characteristics when we move to other types of imaging data with perhaps more detailed sample information.  It also provides a description of the experimental variables or conditions we have seen so far such as environmental stress, compound treatment, media and genotype.
```

Comment:
Not a fan of the repeated use of 'so far' in this paragraph, don't think you really need either of them given the context and certainly the first one doesn't read very nicely.